tripal_chado/src/Plugin/TripalImporter/TaxonomyImporter.php

```diff
@@ -55,6 +55,16 @@ class TaxonomyImporter extends ChadoImporterBase implements ContainerFactoryPlug
    */
   protected object $property_buddy;
 
+  /**
+   * Options for file retrieval from NCBI
+   *
+   * @var array
```

Comment:
Same as my suggestions in the pub med importer.

```suggestion
   * @var array
   *   Options to be passed to the file retrieval service.
   *   @see Drupal\tripal\Services\TripalFileRetriever::retrieveFileContents()
```